README.md

```diff
@@ -79,6 +79,10 @@ working with Memory, and see how the kernel works.
      [Polyglot extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode)
      to run the notebooks.
 
+### Working with Secrets
```

Comment:
None of these projects use *.config files. I believe a better place for this comment would be in the KernelHttpServer README and perhaps just have a reference to that README in these files.

Reply:
done